proposals/csharp-12.0/collection-expressions.md

```diff
@@ -219,7 +213,37 @@ ImmutableArray<int> ia =
 ## Construction
 [construction]: #construction
 
-*Give specific ordering for determining how to construct the constructible collection types.*
+`Length` and `Count` properties, and `GetEnumerator` methods, are assumed to have no side effects.
+
+If the target type is a *struct* or *class type* that implements `System.Collections.IEnumerable`, and the target type does not have a *[create method](#create-methods)*, the construction of the collection instance as follows:
```

Comment:
Missing a word at the end of the sentence; either a "proceeds as follows", or "occurs", or some other phrasing of your choice.